tests/Oryx.Integration.Tests/Node/NodeAngularAppTest.cs

```diff
@@ -68,6 +68,73 @@ public async Task CanBuildAndRun_Angular6NodeApp_WithoutZippingNodeModules(strin
                 });
         }
 
+        [Theory]
+        [InlineData("8")]
+        [InlineData("9.4")]
+        public async Task CanBuildAndRun_Angular6NodeApp_Multiple_Restart_WithoutZippingNodeModules(string nodeVersion)
+        {
+            // Arrange
+            var appName = "angular6app";
+            var volume = CreateAppVolume(appName);
+            var appDir = volume.ContainerDir;
+            var buildScript = new ShellScriptBuilder()
+               .AddCommand($"oryx build {appDir} --platform nodejs --platform-version {nodeVersion}")
```

Comment:
Run an `npm install` on the `appDir` before this line so that it creates the `node_modules` dir. This will invoke the condition https://github.com/microsoft/Oryx/pull/531/files#diff-151045f085135b054c71fb70505f1d97R115

Similarly in the other test too

Reply:
as discussed .. this needs to be in run_script and not in build script .. updated tests